test/src/test/java/hudson/security/HudsonPrivateSecurityRealmFIPSTest.java

```diff
@@ -25,12 +25,8 @@
 package hudson.security;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.hasItem;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.not;
-import static org.hamcrest.Matchers.startsWith;
+import static org.hamcrest.Matchers.*;
```

Comment:
Do not use wildcards in the imports. Add as many imports as you need

Reply:
done